src/Randomizer.SMZ3.Tracking/locations.json

```diff
@@ -504,19 +504,26 @@
       "Name": [ "Crocomire" ],
       "Image": "stoke.png",
       "Column": 8,
-      "Row": 5
+      "Row": 5,
+      "WhenDefeated": [
+        "Toggled Crocomire on.",
+        "Toggled Crocomire on. Did you actually have to kill him this time?",
+        [ "Toggled Crocomire on. It looks like you brought more than three super missles. Good job.", 0.1 ]
```

Comment:
Is "missles" here a typo or intentional?

Reply:
I asked Fragger about another typo I noticed, and it was intentional to have tracker pronounce things in a more desirable way. However, I just tested this one and it sounds the same to me either way, so I corrected the typo.